js/exportsms.js

```diff
@@ -355,13 +374,34 @@ var exportsms = function() {
           choice = configObject.choices[j];
           if (endLevelGroupSuccessFailureStatus == choice.flag.toUpperCase() && !choice.next) {
             choice.next = parseInt(passageDatum.optinpath, 10);
-            partialStory[nameString] = configObject;
             break;
           }
         }
+
+        nextLevelNumber = parseInt(levelNumber, 10) + 1
+        if (nextLevelNumber <= TOTAL_NUMBER_OF_LEVELS) {
+          var nextLevelString = 'L' + nextLevelNumber + '0'
```

Comment:
semicolon and maybe declare `var nextLevelString` at the top of the function with the rest of the vars.